ddl/serial_test.go

```diff
@@ -84,6 +84,7 @@ func (s *testSerialSuite) TearDownSuite(c *C) {
 	if s.store != nil {
 		s.store.Close()
 	}
+	s.mvccStore.Close()
```

Comment:
Could we move line68-69 to `TestMultiRegionGetTableEndHandle` before [line160](https://github.com/pingcap/tidb/blob/master/ddl/serial_test.go#L160)? `cluster` and `mvccStore` only use in this test.